package.json

```diff
@@ -126,7 +126,7 @@
   "dependencies": {
     "@dha-team/arbundles": "^1.0.1",
     "@permaweb/aoconnect": "^0.0.57",
-    "arweave": "1.14.4",
+    "arweave": "1.15.5",
```

Comment:
this always makes me nervous. will do some validations across our clients (observer, ar-io-node, network-portal, arns.app) to validate if it causes any issues

Reply:
minor version bump -- "Drop Node 16 Support". that semantic versioning 😂